src/main/java/UofTeams.java

```diff
@@ -0,0 +1,233 @@
+import view_comment.ui.ViewCommentView;
+import view_comment.use_case.ViewCommentDsGateway;
+import view_comment.use_case.ViewCommentInputBoundary;
+import view_comment.use_case.ViewCommentInteractor;
+import view_comment.use_case.ViewCommentOutputBoundary;
+import view_post.drivers.ViewPostDatabaseAccess;
+import view_post.interface_adapters.ViewPostController;
+import view_post.interface_adapters.ViewPostPresenter;
+import view_post.interface_adapters.ViewPostViewModel;
+import view_post.ui.*;
+import view_post.use_case.ViewPostDsGateway;
+import view_post.use_case.ViewPostInteractor;
+
+import java.util.Arrays;
+
+/*
+* The main class for the file, containing the main method which runs the program
+* */
+public class UofTeams {
+    static String adminPath = new String("src/main/java/Database/admin.csv");
+    static String numPostsCreatedFilePath = new String("src/main/java/Database/numPostsCreated.csv");
+    static String numUsersCreatedFilePath = new String("src/main/java/Database/numUsersCreated.csv");
+    static String numCommentsCreatedFilePath = new String("");
+    static String postsFilePath = new String("src/main/java/Database/posts.csv");
+    static String usersFilePath = new String("src/main/java/Database/users.csv");
+    static String commentsFilePath = new String("src/main/java/Database/comments.csv");
+    static String generalPath = new String("src/main/java/Database/");
+
+    public static void main(String[] args) {
+        // initialize stuff for delete_post
+        DeletePostViewModel deletePostViewModel = new DeletePostViewModel();
+        DeletePostOutputBoundary deletePostPresenter = new DeletePostPresenter(deletePostViewModel);
+        delete_post.use_case.PostReaderInterface deletePostFactory = new delete_post.use_case.PostFactory();
+        DeletePostDsGateway deletePostDataAccess = new DeletePostDataAccess(generalPath, deletePostFactory);
+        DeletePostInputBoundary deletePostInteractor = new DeletePostInteractor((DeletePostPresenter) deletePostPresenter, deletePostDataAccess);
+        DeletePostController deletePostController = new DeletePostController(deletePostInteractor);
+        DeleteView deleteView = new DeleteView(deletePostController);
+
+        // initialize stuff for filter_post
+        FilterPostViewModel filterPostViewModel = new FilterPostViewModel(new String[0], new int[0], new String[0]);
+        FilterPostOutputBoundary filterPostPresenter = new FilterPostPresenter(filterPostViewModel);
+        FilterPostDsGateway filterPostDataAccess = new FilterPostDataAccess(postsFilePath);
+        FilterPostInputBoundary filterPostInteractor = new FilterPostInteractor(filterPostDataAccess, filterPostPresenter);
+        FilterPostController filterPostController = new FilterPostController(filterPostInteractor);
+        // Requires array of preset tags
+        FilterPostBarView filterPostBarView = new FilterPostBarView(new String[]{"sports", "quantum", "math"}, filterPostController);
```

Comment:
Just leaving a comment here as a reference: we need to come up with a list of tags and update that here.